data-fetching/axios.md

```diff
@@ -0,0 +1,81 @@
+# Axios
+
+* [General information](#general-information)
+* [Pros and cons](#pros-and-cons)
+* [Configuration](#configuration)
+
+## General information
+
+Axios is a promise-based HTTP Client for node.js and the browser. It is isomorphic (= it can run in the browser and nodejs with the same codebase). On the server-side it uses the native node.js http module, while on the client (browser) it uses XMLHttpRequests.
+
+The official documentation you can find [here](https://axios-http.com/docs/intro).
+
+ ---
+
+## Pros and cons
+
+Pros:
+
+1. **Simplicity**: Axios provides a simple and straightforward API, making it easy to use and understand. It abstracts away the complexities of making HTTP requests, handling responses, and managing errors.
+
+2. **Flexibility**: Axios can be used in various environments, including web browsers, React Native, and Node.js. It supports different platforms and provides consistent functionality across them.
+
+3. **Promise-based**: Axios uses promises, which allows you to handle asynchronous operations in a more elegant and readable way. Promises enable chaining of operations, error handling, and better control flow.
+
+4. **Request and response interception**: Axios allows you to intercept and modify requests and responses through the use of interceptors. This feature is helpful for adding custom headers, handling authentication, and performing other tasks before the request is sent or after the response is received.
+
+5. **Automatic request and response transformation**: Axios can automatically transform request and response data. For example, it can parse JSON responses into JavaScript objects or convert request payloads into different formats like JSON, FormData, or URL-encoded data.
```

Comment:
I would add one more:
* It's easy to mock, there are a lot of libraries to mock and use mocks with the tests